### PR TITLE
ref(sidebar): Add expandable accordion & use for Starfish

### PR DESCRIPTION
**Pull request by vuluongj20**
**Before ——**
image
image

**After ——**
image
image
image

### CODE REVIEW

**Review by evanpurkhiser — APPROVED** [review]
Code looks good

File: static/app/components/sidebar/index.tsx [hunk 1]
```diff
@@ -227,35 +228,37 @@ function Sidebar({location, organization}: Props) {
...
-        {...sidebarItemProps}
-        icon={<IconStar />}
-        label={<GuideAnchor target="starfish">{t('Database')}</GuideAnchor>}
-        to={`/organizations/${organization.slug}/starfish/database/`}
-        id="starfish"
-      />
+      >
+        <SidebarItem
+          {...sidebarItemProps}
+          label={<GuideAnchor target="starfish">{t('Api')}</GuideAnchor>}
```
Comment (gggritso) [inline static/app/components/sidebar/index.tsx:242]:
```suggestion
          label={<GuideAnchor target="starfish">{t('API')}</GuideAnchor>}
```
✨

File: static/app/components/sidebar/sidebarAccordion.tsx [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+const SidebarAccordionHeaderWrap = styled('div')`
+  position: relative;
+  display: flex;
+  & > *:first-child {
+    width: 100%;
+    flex-shrink: 1;
+  }
+`;
+
+const SidebarItemExandButton = styled(Button)<{sidebarCollapsed?: boolean}>`
```
Comment (gggritso) [inline static/app/components/sidebar/sidebarAccordion.tsx:83]:
```suggestion
const SidebarItemExpandButton = styled(Button)<{sidebarCollapsed?: boolean}>`
```
Tiny typo here

**Review by gggritso — APPROVED** [review]
This is _awesome_, thanks, Vu! One nit, one typo, but this looks _super great_

**Pull request merged**